pkg/apiserver/apiserver.go

```diff
@@ -54,7 +54,7 @@ var (
 			Name: "apiserver_request_count",
 			Help: "Counter of apiserver requests broken out for each request handler, verb, API resource, and HTTP response code.",
```

Comment:
Please update the help text to include the new field (and remove handler if you stick with removing it)